src/pages/en/about/teams.mdx

```diff
@@ -115,232 +126,194 @@ The Chasm team is responsible for maintaining Chasm, the Collision Handling ASM
 
 This team is responsible for the following projects:
 
-<div class="button-grid">
-	<ProjectButton name="asmr-processor-prototype" />
-	<ProjectButton name="chasm" />
-</div>
+<ProjectButtons names={["asmr-processor-prototype", "chasm"]} />
 
-<div class="developer team-grid">
-	<TeamMembers
-		list={["CheaterCodes", "Earthcomputer", "Kroppeb", "Pyrofab"]}
-	/>
-</div>
+<TeamMembers
+	type="developer"
+	list={["CheaterCodes", "Earthcomputer", "Kroppeb", "Pyrofab"]}
```

Comment:
This seems to be missing @FoundationGames, based on the `Team: Chasm` role in the Toolchain discord.

Reply:
I asked Foundation for card information back in September, they still haven't responded.